src/wp-admin/css/forms.css

```diff
@@ -1576,14 +1595,24 @@ table.form-table td .updated p {
 	}
 
 	.form-table.permalink-structure td code {
-		margin-left: 32px;
 		display: inline-block;
 	}
 
+	.form-table.permalink-structure .structure-selection {
+		margin-top: 8px;
+	}
+
+	.form-table.permalink-structure .structure-selection .row > div {
+		max-width: calc(100% - 36px);
+		width: 100%;
+	}
+
+	.form-table.permalink-structure .structure-selection .row label {
+		font-weight: 700;
```

Comment:
I already said "one more" change, but these labels had a `font-weight` of 600 at any screen size when they were in the separate column. I recommend retaining the 600 value and moving this style earlier in the CSS so it affects all screen sizes.

Reply:
With 600 as `font-weight` on all screen sizes the layout will look like this. Commit is incoming. 
<img width="928" alt="bild" src="https://user-images.githubusercontent.com/11491369/173183284-e78b42fd-5bd0-49a7-b68f-f95a9791d398.png">